### PR TITLE
Wri

### PR DESCRIPTION
**Pull request by yeah800**
all new

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -208,6 +208,53 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their Guidelines \n- Contracts signed \n- literature wrote \n- Press Mentions",
```
Comment (reasn) [inline data.js]: Removed caps in sentences: `"- Organizations using their guidelines \n- Contracts signed \n- literature wrote \n- Press mentions",`

File: data.js [hunk 2]
```diff
@@ -208,6 +208,53 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Their outcome is measured well. These are some of (2018)[https://www.wri.org/annualreport/2018-19#page=4] and their are (more)[https://www.wri.org/our-work/outcomes].",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+
+        },
+        activeCountries: {
+            content: "Offices (global)[https://www.wri.org/annualreport/2018-19] - Headquarter is in Washington, DC.",
```
Comment (reasn) [inline data.js]: "They have [offices globally][https://www.wri.org/annualreport/2018-19], their headquarter is in Washington, DC."

File: data.js [hunk 3]
```diff
@@ -208,6 +208,53 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        yearlyIncome: {
+            content: "(~ $ 111M in 2018)[https://wriorg.s3.amazonaws.com/s3fs-public/uploads/wri-2018-19-annual-report_0.pdf#page=48]",
```
Comment (reasn) [inline data.js]: `()[]` -> `[]()`

File: data.js [hunk 4]
```diff
@@ -208,6 +208,53 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        members: {
+            content: "over (1000 Scientists and staff members)[https://www.wri.org/about/experts-staff]",
```
Comment (reasn) [inline data.js]: `scientists` (lowercase)

File: data.js [hunk 5]
```diff
@@ -208,6 +208,53 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        employees: {
```
Comment (reasn) [inline data.js]: would leave that out or make clear why it's unclear

File: data.js [hunk 6]
```diff
@@ -208,6 +208,53 @@ exports.data = [
...
+            content: "unclear",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        participation: {
```
Comment (reasn) [inline data.js]: 3 empty fields

**Review by reasn — CHANGES_REQUESTED** [review]

File: data.js (hunk 4 above)
Comment (yeah800) [reply data.js]: done

File: data.js (hunk 2 above)
Comment (yeah800) [reply data.js]: done but with the right comibnation of () & []

File: data.js (hunk 3 above)
Comment (yeah800) [reply data.js]: done

File: data.js (hunk 6 above)
Comment (yeah800) [reply data.js]: deleted

File: data.js (hunk 5 above)
Comment (yeah800) [reply data.js]: deleted

File: data.js (hunk 1 above)
Comment (yeah800) [reply data.js]: did not get this haha

**Review by yeah800 — COMMENTED** [review]
all done

File: data.js (hunk 1 above)
Comment (reasn) [reply data.js]: if you compare my suggestion and the original, you'll see less capitals.

Comment (reasn) [reply data.js]: @yeah800

**Review by reasn — CHANGES_REQUESTED** [review]
great, one thing left

File: data.js (hunk 1 above)
Comment (yeah800) [reply data.js]: check

**Review by yeah800 — COMMENTED** [review]
All done!

File: data.js [hunk 7]
```diff
@@ -208,6 +208,37 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Their outcome is measured well. These are some of (2018)[https://www.wri.org/annualreport/2018-19#page=4] and their are (more)[https://www.wri.org/our-work/outcomes].",
```
Comment (reasn) [inline data.js]: wrong brackets. Also, for future reference: the format we're using is called Markdown. It's very common in software engineering (but sadly only there -.-):
https://duckduckgo.com/?q=markdown+cheat+sheet

**Review by reasn — CHANGES_REQUESTED** [review]
one issue with the hyperlink syntax

File: data.js (hunk 7 above)
Comment (reasn) [reply data.js]: @yeah800 tip for future PRs (pull requests): Read it through yourself for common & easy mistakes to reduce back-and-forth

Comment (yeah800) [reply data.js]: fixed the link

File: data.js [hunk 8]
```diff
@@ -208,6 +208,37 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- literature wrote \n- Press mentions",
```
Comment (reasn) [inline data.js]: `Literature wrote` seems wrong. `Articles written` / `Papers written` ?

File: data.js [hunk 9]
```diff
@@ -208,6 +208,37 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Their outcome is measured well. These are some of [2018](https://www.wri.org/annualreport/2018-19#page=4) and their are (more)[https://www.wri.org/our-work/outcomes].",
```
Comment (reasn) [inline data.js]: The language seems a bit off (`There are some of 2018 and their are more`). But more general I think the `actions` field should contain a description of their actions, not just links to reports or their website.

**Review by reasn — CHANGES_REQUESTED** [review]

File: data.js (hunk 9 above)
Comment (yeah800) [reply data.js]: put some real actions in there

File: data.js (hunk 8 above)
Comment (yeah800) [reply data.js]: fixed

**Comment by yeah800** [comment]
all right

File: data.js [hunk 10]
```diff
@@ -208,6 +208,41 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- Articles wrote \n- Press mentions",
```
Comment (reasn) [inline data.js]: Articles wrote is still no English, see previous comments

File: data.js [hunk 11]
```diff
@@ -208,6 +208,41 @@ exports.data = [
...
+            updatedBy: "Andre", updatedAt: "2020-02-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Examples of [2018](https://www.wri.org/annualreport/2018-19#page=4):"+
```
Comment (reasn) [inline data.js]: Examples *from*

File: data.js [hunk 12]
```diff
@@ -208,6 +208,41 @@ exports.data = [
...
+                     "\n- Guidlines of WRI were adopted by several cities to develope a more sustainable traffic."+ 
+                     "\n- Were lobbying for the Paris Agreement."+
```
Comment (reasn) [inline data.js]: Would remove `Were `, the context makes clear that it's in the past

File: data.js [hunk 13]
```diff
@@ -208,6 +208,41 @@ exports.data = [
...
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Examples of [2018](https://www.wri.org/annualreport/2018-19#page=4):"+
+                     "\n- Guidlines of WRI were adopted by several cities to develope a more sustainable traffic."+
```
Comment (reasn) [inline data.js]: 1. develop*e*
2. This is urban mobility, traffic is what you're stuck in
2. Suggestion `Creating guidelines for [what?] (adopted by [more precise, big, small, worldwide, European, ???] cities to make urban mobility more sustainable)`

File: data.js [hunk 14]
```diff
@@ -208,6 +208,41 @@ exports.data = [
...
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Examples of [2018](https://www.wri.org/annualreport/2018-19#page=4):"+
+                     "\n- Guidlines of WRI were adopted by several cities to develope a more sustainable traffic."+ 
+                     "\n- Were lobbying for the Paris Agreement."+
+                     "\n- Helped to implement first binding environmental laws in 16 South American states."+
```
Comment (reasn) [inline data.js]: South America has 13 countries. If this is referring to states it seems necessary to mention of which countries they are. It differs from country to country what a state means (e.g. France is not really federal)

File: data.js [hunk 15]
```diff
@@ -208,6 +208,41 @@ exports.data = [
...
+                     "\n Read about (more)[https://www.wri.org/our-work/outcomes] stories of success though out the past years.",
```
Comment (reasn) [inline data.js]: 1. `Read more stories`, not `Read about more stories`
2. though out -> you probably mean throughout but "stories from the past would be the correct version. I suggest to leave out, all stories are from the past.

**Review by reasn — CHANGES_REQUESTED** [review]
Not sure it got better. Added comments

File: data.js (hunk 10 above)
Comment (yeah800) [reply data.js]: deleted completely. That was a bit of a useless kpi

File: data.js (hunk 15 above)
Comment (yeah800) [reply data.js]: done

File: data.js (hunk 14 above)
Comment (yeah800) [reply data.js]: researched better

File: data.js (hunk 13 above)
Comment (yeah800) [reply data.js]: changed wording

File: data.js (hunk 12 above)
Comment (yeah800) [reply data.js]: changed wording

File: data.js (hunk 11 above)
Comment (yeah800) [reply data.js]: ok

File: data.js [hunk 16]
```diff
@@ -208,6 +208,41 @@ exports.data = [
...
+        activeCountries: {
+            content: "They have [offices globally](https://www.wri.org/annualreport/2018-19), their headquarter is in Washington, DC.",
+            updatedBy: "Andre", updatedAt: "2020-01-21",
+        },
+        yearlyIncome: {
+            content: "[~ $ 111M in 2018](https://wriorg.s3.amazonaws.com/s3fs-public/uploads/wri-2018-19-annual-report_0.pdf#page=48)",
+            updatedBy: "Andre", updatedAt: "2020-01-21",
+        },
+        members: {
+            content: "over [1000 scientists and staff members](https://www.wri.org/about/experts-staff)",
```
Comment (reasn) [inline data.js:243]: Uppercase `O`. Please double-check your PRs everytime. Reviewers tend to miss a lot, too.

**Review by reasn — CHANGES_REQUESTED** [review]

File: data.js (hunk 16 above)
Comment (reasn) [reply data.js:243]: @yeah800 I updated crisisfighters.org with your changes, including the `O`. We can close this now and have the new content online :)

**Pull request closed without merging**